source/install/mattermost-kubernetes-operator.rst

```diff
@@ -180,6 +180,141 @@ The process described below needs to be completed prior to proceeding with the M
   The deployment process can be monitored in the Kubernetes user interface. If errors or issues are experienced, review the Mattermost, Operator, and MySQL logs for guidance including error messages. If remediation is not successful, contact Mattermost customer support for assistance.
 
   Once complete, access your Mattermost instance and confirm that the database has been restored.
+Migrate From Helm Chart
+--------------------------------------------
+
+If you have previosuly installed mattermost as helm chart deployment, you can easily move to an operator installed following the provided steps
+
+1. Create a namespace dedicated fo the operator
+
+.. code-block:: sh
+
+  $ kubectl create ns mattermost-operator
+
+2. Save the secrets from the current installation , if you have extra secrets mounted as a volume save them as well
+
+.. code-block:: sh
+
+  $ kubectl get secrets -n mattermost mattermost-db-secret -o yaml > mattermost-db-secret.yaml
```

Comment:
To make it easier to follow, we could export namespace as env var:

```suggestion
2. Save the secrets from the current installation, if you have extra secrets mounted as a volume save them as well. To begin, set ```NAMESPACE``` environment variable to the namespace your Mattermost instance is running in:

.. code-block:: sh
  $ export NAMESPACE=mattermost
  $ kubectl get secrets -n $NAMESPACE mattermost-db-secret -o yaml > mattermost-db-secret.yaml
```
We would need to also adjust other commands.